src/content/basics/security/index.md

```diff
@@ -51,3 +45,65 @@ kubelet data is stored under `/var/lib/kubelet`, Docker images under `/var/lib/d
+
+## Infrastructure and Network Security
+
+### Physical Security
+
+#### Public Cloud Providers
+
+Our customers running on a public cloud enjoy the hardening offered by their provider in order to ensure the physical security of their data-centers.
+This typically includes perimeter security, CCTV, vetted security staff, and logged biometric access control, among others.
+For more details, visit the providers' websites:
+
+[AWS](https://aws.amazon.com/compliance/data-center/controls)
+
+[Azure](https://docs.microsoft.com/en-us/azure/security/fundamentals/physical-security)
+
+#### Private Cloud / On-Premises
+
+Customers may also run the Giant Swarm platform within their own data centers, which they have hardened to their satisfaction.
+
+### Security Testing
+
+#### Conformance Testing
+
+We regularly test our configurations against the CIS Kubernetes Benchmark as well as other CIS and industry benchmarks for Docker, Linux, and our cloud providers to ensure our platform remains compliant with industry best-practices. Giant Swarm's Kubernetes platform is also [CNCF certified](https://www.cncf.io/certification/software-conformance/).
+
+#### Penetration Testing
+
+Giant Swarm customers regularly run penetration tests on their applications, running on our platform. Findings from these tests which the customer chooses to share with us are used to improve the overall platform security.
+
+#### Compliance Audits
+
+[AWS](https://aws.amazon.com/compliance/soc-faqs) and [Azure](https://docs.microsoft.com/en-us/microsoft-365/compliance/offering-soc?view=o365-worldwide) are compliant with SOC 1,2,3 as well as PCI DSS, GDPR and other compliance frameworks. See the provider pages for details.
+
+### Kubernetes Security
+
+Regardless of provider, clusters start with restrictive settings already in place. All internal and external network traffic is denied by default, role-based access control prevents unauthorized access to Kubernetes resources, and Pod security policies are in place to prevent Pods from running with root users, risky volume configurations, or additional privileges. All of these can be adapted by the customer to fit their needs.
+
+### Network Security
+
+Each cluster is completely isolated, running inside its own VPC (AWS) or Virtual Network (Azure) within your account. They can be further isolated by keeping clusters in different accounts.
+
+On-premises we achieve similar isolation by running the nodes in hypervisors (KVM, VMWare) and isolating clusters' networks within separate VXLAN bridges. We encourage separation of workloads over several clusters to limit the blast radius of incidents.
+
+## Authentication
+
+### Management Access
+
+Giant Swarm needs access to your clusters in order to provide day 2 support. For details on how we protect this access please visit the [Secure access to clusters - Users and Giant Swarm support](../secured-access-to-clusters/index.md) page.
```

Comment:
```suggestion
Giant Swarm needs access to your clusters in order to provide day 2 support. For details on how we protect this access please visit the [Secure access to clusters - Users and Giant Swarm support](/basics/secured-access-to-clusters/) page.
```

For internal links in docs:

- We use absolute paths (starting with `/` instead of `../`).
- We don't set any file names in the end like `index.md`, as they won't appear in the URLs of the final HUGO rendered site.